src/cid.js

```diff
@@ -78,9 +78,10 @@ export class CID {
     /** @readonly */
     this.bytes = bytes
 
-    // Circular reference
+    // flag to serializers that this is a CID and
+    // should be treated specially
     /** @readonly */
-    this.asCID = this
+    this['/'] = bytes
   }
 
```

Comment:
I think it would be a good idea to keep `asCID` getter around as there is a code in the wild that will break otherwise. This is not going to help with moving cids across realms, but at least it will work in other instances.

```suggestion

/**
 * Signalling `cid.asCID === cid` got replaced by `cid['/'] === cid.bytes` please
 * either use `CID.asCID(cid)` or switch to new signalling mechanism.
 * @deprecated
 */
get asCID() {
  return this
}
```

Reply:
Done!